sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/src/Generated/Models/TaskStepProperties.cs

```diff
@@ -18,7 +18,6 @@ namespace Microsoft.Azure.Management.ContainerRegistry.Models
     /// <summary>
     /// Base properties for any task step.
     /// </summary>
-    [Newtonsoft.Json.JsonObject("TaskStepProperties")]
     public partial class TaskStepProperties
```

Comment:
No swagger change caused this diff.  To me this means I am using a different autorest version from the previous author. To avoid random change, please let me know what version I should use. Right now I have 
```
C:\sdk\azure-sdk-for-net>autorest --version
AutoRest code generation utility [version: 2.0.4413; node: v10.19.0]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest


Showing All Installed Extensions

 Type       Extension Name                           Version      Location
 core       @microsoft.azure/autorest-core           2.0.4302     C:\Users\yugangw\.autorest\@microsoft.azure_autorest-core@2.0.4302
 core       @microsoft.azure/autorest-core           2.0.4413     C:\Users\yugangw\.autorest\@microsoft.azure_autorest-core@2.0.4413
 extension  @microsoft.azure/autorest.csharp         2.3.82       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.csharp@2.3.82
 extension  @microsoft.azure/autorest.modeler        2.3.44       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.modeler@2.3.44
 extension  @microsoft.azure/autorest.modeler        2.3.55       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.modeler@2.3.55
 extension  @microsoft.azure/autorest.python         3.0.62       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.python@3.0.62

```

Reply:
Again, the generate.ps1 installs the autorest version it requires, hence there is nothing I can change